virttest/utils_stress.py

```diff
@@ -86,8 +113,51 @@ def vm_stress_events(self, event, vm):
                                 'cur_config': self.current_vcpu,
                                 'cur_live': self.current_vcpu,
                                 'guest_live': self.current_vcpu}
-                    utils_hotplug.check_vcpu_value(vm, exp_vcpu, option="--live")
+                    utils_hotplug.check_vcpu_value(
+                        vm, exp_vcpu, option="--live")
             elif "reboot" in event:
                 vm.reboot()
+            elif "nethotplug" in event:
+                for i in range(int(self.iface_num)):
+                    logging.debug("Try to attach interface %d" % i)
+                    mac = utils_net.generate_mac_address_simple()
+                    options = ("%s %s --model %s --mac %s %s" %
+                               (self.iface_type, self.iface_source['network'],
+                                self.iface_model, mac, self.attach_option))
+                    logging.debug("options for attach: %s" % options)
+                    ret = virsh.attach_interface(vm.name, options,
+                                                 ignore_status=True)
+                    time.sleep(self.event_sleep_time)
+                    libvirt.check_exit_status(ret)
+                    if self.detach_option:
+                        options = ("--type %s --mac %s %s" %
+                                   (self.iface_type, mac, self.detach_option))
+                        logging.debug("options for detach: %s" % options)
+                        ret = virsh.detach_interface(vm.name, options,
+                                                     ignore_status=True)
+                        libvirt.check_exit_status(ret)
+            elif "diskhotplug" in event:
+                for i in list(range(len(self.device_source_names))):
```

Comment:
range will return `list` so, casting is not necessary

Reply:
same here